Source/Widgets/Viewer/viewerDragDropMixin.js

```diff
@@ -150,6 +152,20 @@ define([
             },
 
             /**
+             * Gets or sets a value indicating if the camera should fly to the data source after it is loaded.
+             * @memberof viewerDragDropMixin.prototype
+             * @type {Boolean}
+             */
+            flyToOnDrop : {
+                get : function() {
+                    return flyToOnDrop;
+                },
+                set : function(value) {
+                    flyToOnDrop = value;
```

Comment:
It looks like proxy and clearOnDrop are simple get/set as well, so I was most likely just following suit.  I'm not sure why they are though, so I'll change them and see what happens.